pylintrc_tweaks

```diff
@@ -2,3 +2,6 @@
 [MASTER]
 ignore+ = ,input
 load-plugins = edx_lint.pylint
+
+[MESSAGES CONTROL]
+disable+ = ,unicode-format-string
```

Comment:
This might be controversial.

Reply:
What does this particular flag do? Disable this check by default?

Reply:
This disables it for this repo's own pylint checks.  It's on by default for repos making use of edx-lint.

Reply:
Any particular reason to disable it here?

Reply:
I knew it would be controversial :)   No reason, other than this repo doesn't traffic in non-ASCII data.  I'll leave it enabled and fix the violations.